Problem Installing gem

HI,

I tried a straight install as per the readme but I got a compilation error. `core_bluetooth.m:7:9: fatal error: 'CoreBluetooth/CoreBluetooth.h' file not found`

I've googled a bit and there are hints that 'CoreBluetooth seem to only be includable through IOBluetooth'  ( http://lists.apple.com/archives/bluetooth-dev/2013/May/msg00011.html ) but I've been unable to figure out how to apply this to the install.

Can you help at all ?

Thanks

I have a problem at `rake compile`. I think the key error is `ld: framework not found CoreBluetooth`.  

The full set of messages is :
```
$ rake compile
cd tmp/x86_64-darwin14/core_bluetooth/2.2.3
/usr/bin/make
compiling ../../../../ext/core_bluetooth/core_bluetooth.m
../../../../ext/core_bluetooth/core_bluetooth.m:27:243: warning: declaration of 'struct BluetoothDeviceAddress' will not be visible outside of this
      function [-Wvisibility]
  ...advertisingType:(unsigned char)arg3 ownAddressType:(unsigned char)arg4 directAddressType:(unsigned char)arg5 directAddress:(struct BluetoothDevic...
                                                                                                                                        ^
../../../../ext/core_bluetooth/core_bluetooth.m:30:27: warning: declaration of 'struct BluetoothDeviceAddress' will not be visible outside of this
      function [-Wvisibility]
- (int)getAddress:(struct BluetoothDeviceAddress { unsigned char x1[6]; }*)arg1;
                          ^
2 warnings generated.
linking shared-object scan_beacon/core_bluetooth.bundle
ld: warning: directory not found for option '-L/Users/haven/.sm/pkg/active/lib'
ld: framework not found CoreBluetooth
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [core_bluetooth.bundle] Error 1
rake aborted!
Command failed with status (2): [/usr/bin/make...]
/Users/andy/.rvm/gems/ruby-2.2.3/gems/rake-compiler-0.9.9/lib/rake/extensiontask.rb:157:in `block (2 levels) in define_compile_tasks'
/Users/andy/.rvm/gems/ruby-2.2.3/gems/rake-compiler-0.9.9/lib/rake/extensiontask.rb:156:in `block in define_compile_tasks'
/Users/andy/.rvm/gems/ruby-2.2.3/bin/ruby_executable_hooks:15:in `eval'
/Users/andy/.rvm/gems/ruby-2.2.3/bin/ruby_executable_hooks:15:in `<main>'
Tasks: TOP => compile => compile:x86_64-darwin14 => compile:core_bluetooth:x86_64-darwin14 => copy:core_bluetooth:x86_64-darwin14:2.2.3 => tmp/x86_64-darwin14/core_bluetooth/2.2.3/core_bluetooth.bundle
(See full trace by running task with --trace)
```

--- a/ext/core_bluetooth/core_bluetooth.m
+++ b/ext/core_bluetooth/core_bluetooth.m
@@ -4,7 +4,6 @@
 #include "ruby.h"
 #import <Foundation/Foundation.h>
 #import <IOBluetooth/IOBluetooth.h>
-#import <CoreBluetooth/CoreBluetooth.h>
 
 // Defining a space for information and references about the module to be stored internally
 VALUE cb_module = Qnil;

--- a/ext/core_bluetooth/extconf.rb
+++ b/ext/core_bluetooth/extconf.rb
@@ -9,7 +9,6 @@ dir_config(extension_name)
 
 if RUBY_PLATFORM =~ /darwin/
   $DLDFLAGS << " -framework Foundation"
-  $DLDFLAGS << " -framework CoreBluetooth"
   $DLDFLAGS << " -framework IOBluetooth"
 else
   # don't compile the code on non-mac platforms because

